lib/client.js

```diff
@@ -158,7 +158,7 @@ Client.prototype = {
     });
 
     /* Custom error handler */
-    if (this.errorHandler) {
+    if (!options.disableErrorHandler && this.errorHandler) {
```

Comment:
This works, but a more flexible approach might be to allow the caller to intercept the error before the global error handler kicks in.

So, something like:

```js
// Install local error handler
if (options.errorHandler) {
  promise = promise.catch(options.errorHandler);
}

// Install global error handler
if (this.errorHandler) {
  promise = promise.catch(this.errorHandler);
}

return promise;
```

This way, you get given the option to swallow the error locally before the global error handler gets called. So, this gives you flexibility to ignore the error, conditionally ignore certain errors or throw a different type of error.